surveys/2021-annual-survey/questions.md

```diff
@@ -911,16 +911,64 @@ Select one:
 - Yes
 - No [`NEXT`](#excluding-rust-what-is-your-experience-with-other-kinds-of-programming-languages)
 
-### If you are employed, which category best describes the domain in which *you* work?
+### Which category best describes the your current employer's industry?
 
-> TODO - server networking, desktop application, mobile application, web application, embedded, etc.
+Select all that apply:
+
+- Advertising
+- Agricultural
+- Aerospace
+- Automotive
+- Energy
+- Computer hardware
+- Consumer software
+- Business software
+- Finance
+- Food
+- Film or Television
+- Gaming
+- Healthcare
+- News media
+- Manufacturing
+- Music
+- Telecommunications
+- Transportation
+- Other (open response)
+
+> **justification**
+>
+> We want to see what industries have what representation in the Rust community.
+> While it's impossible to be precise here, we want to get a general idea.
+> This list is largely adopted from this [Wikipedia article](https://en.wikipedia.org/wiki/Outline_of_industry).
 
 ### Do you write or design software in your work?
 
 - Yes, primarily as an individual contributor (i.e., non-manager).
 - I primarily manage others who do.
 - No [`NEXT`](#excluding-rust-what-is-your-experience-with-other-kinds-of-programming-languages)
 
+### Which categories best describes the tech domain(s) you currently write or design software in?
+
+Select all that apply:
+
+- Blockchain
+- Cloud computing applications
+- Cloud computing infrastructure
+- Desktop computer applications
+- Mobile phone applications
+- Desktop computer or mobile phone infrastructure
+- WebAssembly
+- Embedded devices (with operating systems)
+- Embedded devices (bare metal)
+
```

Comment:
I think we should cover frontend and backend (the latter possibly divided into apps and infra) too

Reply:
That's fair. I think we have categories that cover all of this.